unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWSceneBoundarieShould.cs

```diff
@@ -0,0 +1,66 @@
+using DCL;
+using DCL.Components;
+using DCL.Helpers;
+using DCL.Models;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System.Collections;
+using DCL.Controllers;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityGLTF;
+
+public class BIWSceneBoundarieShould : IntegrationTestSuite_Legacy
```

Comment:
```suggestion
public class BIWSceneBoundarieShould : IntegrationTestSuite
```

`IntegrationTestSuite_Legacy` can be avoided in this case. (It only should be used when needing the `CameraController` or systems that are not in `Environment` and tightly coupled to `InitialScene`.